SmileLab/ruleset.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0"?>
+
+<ruleset name="SmileLab" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+    <description>Coding standard for SmileLab projects.</description>
```

Comment:
Same comment than with phpmd ruleset, i think we should replace "SmileLab projects" by "Magento projects"